runtime/doc/health.txt

```diff
@@ -21,6 +21,23 @@ To run all healthchecks, use: >vim
 <
 Plugin authors are encouraged to write new healthchecks. |health-dev|
 
+                                                                    *g:health*
+g:health  This global variable controls the behavior and appearance of the
+          `health` floating window. It should be a dictionary containing the
+          following optional keys:
+          - `style`: string? Determines the display style of the `health` window.
+                            Set to `'float'` to enable a floating window. Other
+                            styles are not currently supported.
+          - `syntax`: string? Specifies the syntax highlighting to use
```

Comment:
why would this be overridable? checkhealth has its own filetype, people can extend that in the usual ways.